packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -41,6 +42,12 @@ import { useIsSwitchingL2Chain } from './TransferPanelMainUtils'
 import { NonCanonicalTokensBridgeInfo } from '../../util/fastBridges'
 import { tokenRequiresApprovalOnL2 } from '../../util/L2ApprovalUtils'
 
+const onTxError = (error: any) => {
+  if (error.code !== 'ACTION_REJECTED') {
```

Comment:
AFAIK Metamask reject with `error.code` 4001. There is a PR currently with utility function : https://github.com/OffchainLabs/arb-token-bridge/pull/625

Reply:
ping: PR got merged, if you can reuse `isUserRejectedError`? Or do we want ONLY `ACTION_REJECTED` and not 4001?